p2p/muxer/mplex/conn.go

```diff
@@ -0,0 +1,48 @@
+package mplex
```

Comment:
Why do we need to import all this code from the mplex library?

Reply:
@gammazero can answer that as he is the original author